llvm/lib/Target/RISCV/RISCVProcessors.td

```diff
@@ -435,7 +438,11 @@ def VENTANA_VEYRON_V1 : RISCVProcessorModel<"veyron-v1",
                                               TuneZExtHFusion,
                                               TuneZExtWFusion,
                                               TuneShiftedZExtWFusion,
-                                              TuneLDADDFusion]>;
+                                              TuneLDADDFusion]> {
+  let MVendorID = 0x61f;
```

Comment:
This is from https://github.com/qemu/qemu/blob/master/target/riscv/cpu_vendorid.h.